apiserver/cmd/main.go

```diff
@@ -71,7 +71,7 @@ func main() {
 type RegisterHttpHandlerFromEndpoint func(ctx context.Context, mux *runtime.ServeMux, endpoint string, opts []grpc.DialOption) error
 
 func startRPCServer(resourceManager *manager.ResourceManager) {
-	klog.Info("Starting gRPC server")
+	klog.Infof("Starting gRPC server at port %s", *rpcPortFlag)
```

Comment:
It's a general practice to log the configuration for the service, otherwise hard to debug.